docs/avatar/index.html

```diff
@@ -76,7 +76,155 @@ <h2 class="heading-3 head-lvl-3" id="available-components">
 						</li>
 					</ul>
 				</aside>
-				<article>avatar</article>
+				<article><div class="component-description">
+					<p class="justify">
+						avatars are used to show profile images in headers or profile
+						information sections. They can be in multiple sizes. xs &mdash;
+						for extra small, sm &mdash; for small, md &mdash; for medium, lg
+						&mdash; for large, and xl for &mdash; extra large. They can be
+						round or squircle in shape.
+					</p>
+				</div>
+				<div class="component-example">
+					<div class="flex flex-gap-2 wrap flex-center /">
+						<div class="avatar avatar-border">
+							<img
+								src="../../assets/avatars/avatar-1.svg"
+								alt="avatar 1"
+								loading="lazy"
+								class="avatar-round avatar-xs"
+							/>
+						</div>
+						<div class="avatar avatar-border">
+							<img
+								src="../../assets/avatars/avatar-4.svg"
+								alt="avatar 4"
+								loading="lazy"
+								class="avatar-sq avatar-sm"
+							/>
+						</div>
+						<div class="avatar avatar-border">
+							<img
+								src="../../assets/avatars/avatar-2.svg"
+								alt="avatar 2"
+								loading="lazy"
+								class="avatar-round avatar-md"
+							/>
+						</div>
+						<div class="avatar avatar-border">
+							<img
+								src="../../assets/avatars/avatar-3.svg"
+								alt="avatar 3"
+								loading="lazy"
+								class="avatar-sq avatar-lg"
+							/>
+						</div>
+						<div class="avatar avatar-border">
+							<img
+								src="../../assets/avatars/avatar-5.svg"
+								alt="avatar 5"
+								loading="lazy"
+								class="avatar-round avatar-xl"
+							/>
+						</div>
+					</div>
+				</div>
+				<div class="code-example">
+					<pre>
+						<code class="html">
```

Comment:
You can use JS to replace all the '<' with '&lt' in the code example, it's difficult to read the code rn

Reply:
Thanks for helping Krituraj. I have changed the code for it.